src/commands/synthetics/__tests__/cli.test.ts

```diff
@@ -191,12 +191,13 @@ describe('run-test', () => {
         defaultTestOverrides: {
           deviceIds: ['chrome.laptop_large'],
           locations: ['us-east-1'],
-          pollingTimeout: 2,
+          pollingTimeout: 3,
           mobileApplicationVersion: '00000000-0000-0000-0000-000000000000',
           mobileApplicationVersionFilePath: './path/to/application.apk',
         },
-        // TODO SYNTH-12989: Clean up `locations` that should only be part of the testOverrides
+        // TODO SYNTH-12989: Clean up `locations` that should only be part of test overrides
         locations: [],
+        // TODO SYNTH-12989: Clean up `pollingTimeout` in favor of `batchTimeout`
         pollingTimeout: 1,
```

Comment:
Should the value here not be 4? It is 4 in the `config-with-all-keys.json` and this test should just be picking up the config from that file, no? 🤔

Reply:
No, because `batchTimeout` takes the precedence over anything (the deprecated root-level `pollingTimeout`, and the deprecated `pollingTimeout` test override in both `global` and `defaultTestOverrides`):

https://github.com/DataDog/datadog-ci/blob/3a716defe86f3973b695a3826389acebb7662293/src/commands/synthetics/__tests__/config-fixtures/config-with-all-keys.json#L4